composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^7 || ^8.0",
         "ext-json": "*",
-        "psr/log": "^1",
+        "psr/log": "^1 || ^2 || ^3",
```

Comment:
Run `make test-php8` and you will see:
```
Fatal error: Declaration of Smsapi\Client\Curl\SmsapiHttpClient::setLogger(Psr\Log\LoggerInterface $logger) must be compatible with Psr\Log\LoggerAwareInterface::setLogger(Psr\Log\LoggerInterface $logger): void in /app/src/Curl/SmsapiHttpClient.php on line 33
```
To fix that you have to drop PHP 7.0 support. Do that and I will release next major version with your changes.